### PR TITLE
Lua client

### PR DESCRIPTION
**Pull request by albanD**

### CODE REVIEW

File: client/lua/README.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+  * `wall_time` and `step` are handled the same as for `add_scalar_value` for each entry independently.
+
+* `get_scalar_values(name)`
+  * Return a list with one entry for each point added for this scalar plot.
+  * Each entry is a list containing [wall_time, step, value].
+
+* `get_histogram_names()`
+  * Returns a list of string containing the name of all the histogram values in this experiment.
+
+* `add_histogram_value(name, hist, tobuild=false, wall_time=-1, step=-1`
```
Comment (edran) [inline client/lua/README.md]: Forgotten closing brackets. See Line 146 / 150 above as well.

File: client/lua/crayon.lua [hunk 2]
```diff
@@ -0,0 +1,342 @@
+local requests = require("requests")
+local socket = require("socket")
+local json = require('cjson.safe')
+
+local __version__ = "0.4"
+
+local CrayonClient = {}
+local CrayonExperiment = {}
+
+-- CrayonClient class
```
Comment (edran) [inline client/lua/crayon.lua]: Should we not use either `class`, `pl.class` or `classis`?

Comment (albanD) [inline client/lua/crayon.lua]: I wasn't sure which class system to use and frankly none was really needed here given that we have 2 classes.

Comment (edran) [inline client/lua/crayon.lua]: Fair enough.

**Pull request merged**